doc/api/schemas/CodeRepositorySettings.yaml

```diff
@@ -0,0 +1,40 @@
+type: object
+description: Code Repository Settings
+properties:
+    projectUrl:
+        type: string
+    environment:
+        type: string
```

Comment:
What does `environment` refer to exactly? Code should be executable in any environment, as long as we provide containers to run it in

Reply:
it is the env you set. Any, as long as you provide credentials.

Reply:
Still don't understand how this fits with the `CodeRepositorySettings`

Reply:
@thomose Laut Fabian kann das environment weg, ist das so korrekt?

Reply:
Ja